.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 sudo: false
 python:
-    - 2.7
+    - 3.7
 env:
     matrix:
-        - TOX_ENV=py27-django19
-        - TOX_ENV=py27-django110
-        - TOX_ENV=py27-django111
-        - TOX_ENV=py27-flake8
+        - TOX_ENV=py37-django19
+        - TOX_ENV=py37-django110
+        - TOX_ENV=py37-django111
+        - TOX_ENV=py37-flake8
 install:
```

Comment:
If we are planning to merge this to master because it still supports Python 2, I suggest we still test against python 2.7 as well. What do you think @madhulika95b  @somexpert

Reply:
normally I'd say that's fine, but since we're mere months away from Python 2 being completely deprecated, I'm not sure it's worth it. I'm not set on that mindset, but that's where I'm leaning personally

Reply:
I would agree with that if we weren't still running this in Python 2 for now.

Reply:
Maybe not add back to travis then, but maybe tox?

Reply:
Are you suggesting to test it in both python2.7 and python 3.7 ? And also in just travis or in tox as well?

Reply:
There's certainly no harm in doing that, or even in adding them back to Travis.